src/start-slidev.sh

```
@@ -0,0 +1,23 @@
+read choice
+
+# 選択されたフォルダのパスを取得
+selected_folder=$(ls -d slides/*/ | sed -n "${choice}p")
+
+if [ -z "$selected_folder" ]; then
+    echo "Invalid selection"
+    exit 1
+fi
+
+# 選択されたフォルダに移動
+cd "$selected_folder"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)